### ISSUE TITLE
Individual page creation: some required fields don't have the required label

### ISSUE THREAD

**Issue by sepidehshahi** [accessibility-exchange/platform#722]
- [ ] Relevant experiences doesn't say if it's required or optional
- [ ] What languages you are working in doesn't have the required or optional label
